### DISCUSSION TITLE
I'm a verified student but Copilot isn't showing up as free for me?

### DISCUSSION THREAD

**Discussion by sn2979** [community/community#62085] [GitHub Education] [answered]
### Select Topic Area

Question

### Body

I just got verified as a student yesterday, but I don't seem to have access to the Student Developer pack, including Copilot. I'm wondering if there is a way to fix this?

**Comment by IgnacioPalma** [comment]
**¡Hello @sn2979! :wave:** 

After claiming the Student Developer Pack, you can access https://github.com/features/copilot, select the option "Copilot for Individuals" (example image), and you will be informed that you can claim GitHub Copilot for free as a student.

Let me know if this help you! :smile: :heart: 

![image](https://github.com/community/community/assets/81819758/c4642a2e-6400-48e2-a139-2c8fffe8a32e)

**Reply by sn2979** [reply]
This is the thing, whenever I select the copilot for individuals, it keeps asking for payment details, but when I check to make sure I am a verified student, Github says I am. I'm not exactly sure what the issue is. Do you have any other suggestions?

**Reply by IgnacioPalma** [reply, marked as answer]
I see the problem. You still don't have the PRO badge, that means that the benefits of the Student Developer Pack still doesn't reflect on your account; this can take a couple of days, but as soon as you have that PRO badge will mean that you can claim GitHub Copilot.

Other related discussion: #60914

**Reply by sn2979** [reply]
Ah, I see. Thank you!

**Comment by Laohay** [comment]
@IgnacioPalma @sn2979  i am pro status and i got the same issue i cant acces for free at copilot yet i got the student pack developper and pro status , someone can help me please ?

**Reply by IgnacioPalma** [reply]
Hello @Laohay :wave:

Did you follow the steps on [this guide made by Microsoft](https://techcommunity.microsoft.com/t5/educator-developer-blog/step-by-step-setting-up-github-student-and-github-copilot-as-an/ba-p/3736279)?
